src/EA.Weee.Web/Areas/Producer/Mappings/ToViewModel/EditOrganisationDetailsMap.cs

```diff
@@ -69,5 +69,37 @@ private ExternalAddressData MapBusinessAddress(SmallProducerSubmissionData sourc
         {
             return mapper.Map<AddressData, ExternalAddressData>(source.CurrentSubmission.BusinessAddressData);
         }
```

Comment:
@stevelee-civica  You shouldnt make any changes to this file.

Reply:
restored